core/src/databases/database.rs

```diff
@@ -0,0 +1,136 @@
+use serde::Serialize;
+use serde_json::Value;
+
+#[derive(Debug, Serialize)]
+pub struct Database {
+    created: u64,
+    data_source_id: String,
+    database_id: String,
+    internal_id: String,
+    name: String,
+}
+
+impl Database {
+    pub fn new_from_store(
```

Comment:
This is not new from store but new?

Reply:
I only use it from store, but yes could def be called `new` since we only have one method to instantiate

Reply:
Isn't `new_from_store` creating the object in store in other places?